backend/src/server/routes/v3/secret-router.ts

```diff
@@ -157,6 +157,11 @@ export const registerSecretRouter = async (server: FastifyZodProvider) => {
         workspaceSlug: z.string().trim().optional().describe(RAW_SECRETS.LIST.workspaceSlug),
         environment: z.string().trim().optional().describe(RAW_SECRETS.LIST.environment),
         secretPath: z.string().trim().default("/").transform(removeTrailingSlash).describe(RAW_SECRETS.LIST.secretPath),
+        deep: z
```

Comment:
@dangtony98  @maidul98 is deep a good param. I felt something like recursive would be better right. Because deep feels like how much deep